cognite/neat/_issues/_base.py

```diff
@@ -215,207 +208,34 @@ def __eq__(self, other: object) -> bool:
-    """In {sheet_name}, row={row}, column={column}: {msg}. [type={type}, input_value={input}]"""
-
-    extra = "For further information visit {url}"
-
-    sheet_name: str
-    column: str
-    row: int
-    type: str
-    msg: str
-    input: Any
-    url: str | None = None
-
-    @classmethod
-    def from_pydantic_error(
-        cls,
-        error: ErrorDetails,
-        read_info_by_sheet: dict[str, SpreadsheetRead] | None = None,
-    ) -> Self:
-        sheet_name, _, row, column, *__ = error["loc"]
-        reader = (read_info_by_sheet or {}).get(str(sheet_name), SpreadsheetRead())
-        return cls(
-            sheet_name=str(sheet_name),
-            column=str(column),
-            row=reader.adjusted_row_number(int(row)),
-            type=error["type"],
-            msg=error["msg"],
-            input=error.get("input"),
-            url=str(url) if (url := error.get("url")) else None,
-        )
-
-    def as_message(self, include_type: bool = True) -> str:
-        input_str = str(self.input) if self.input is not None else ""
-        input_str = input_str[:50] + "..." if len(input_str) > 50 else input_str
-        output = (
-            f"In {self.sheet_name}, row={self.row}, column={self.column}: {self.msg}. "
-            f"[type={self.type}, input_value={input_str}]"
-        )
-        if self.url:
-            output += f" For further information visit {self.url}"
-        return output
+    ...
 
 
 @dataclass(unsafe_hash=True)
 class NeatWarning(NeatIssue, UserWarning):
     """This is the base class for all warnings used in Neat."""
 
-    @classmethod
-    def from_warning(cls, warning: WarningMessage) -> "NeatWarning":
-        """Create a NeatWarning from a WarningMessage."""
-        return DefaultWarning.from_warning_message(warning)
-
-
-@dataclass(unsafe_hash=True)
-class DefaultWarning(NeatWarning):
-    """{category}: {warning}"""
-
-    extra = "Source: {source}"
+    ...
 
-    warning: str
-    category: str
-    source: str | None = None
 
-    @classmethod
-    def from_warning_message(cls, warning: WarningMessage) -> NeatWarning:
-        if isinstance(warning.message, NeatWarning):
-            return warning.message
-
-        return cls(
-            warning=str(warning.message),
-            category=warning.category.__name__,
-            source=warning.source,
-        )
+class MultiValueError(ValueError):
+    """This is a container for multiple errors.
 
-    def as_message(self, include_type: bool = True) -> str:
-        return str(self.warning)
+    It is used in the pydantic field_validator/model_validator to collect multiple errors, which
+    can then be caught in a try-except block and returned as an IssueList.
 
+    """
 
-T_NeatIssue = TypeVar("T_NeatIssue", bound=NeatIssue)
+    def __init__(self, errors: Sequence[NeatIssue]):
+        self.errors = IssueList(errors)
 
 
-class NeatIssueList(list, Sequence[T_NeatIssue], ABC):
+class IssueList(list, Sequence[NeatIssue]):
```

Comment:
No more generic `NeatIssueList`. This was never used, instead we only have the `IssueList`. This means we can no longer create a list of specific types of issues, but that was never used.